wandb/sdk/launch/builder/kaniko_builder.py

```diff
@@ -36,13 +34,26 @@
     image_tag_from_dockerfile_and_source,
 )
 
+kubernetes = get_module(
+    "kubernetes",
+    required="Kubernetes runner requires the kubernetes package. Please"
+    " install it with `pip install wandb[launch]`.",
+)
+client = get_module(
+    "kubernetes.client",
+    required="Kubernetes runner requires the kubernetes package. Please"
+    " install it with `pip install wandb[launch]`.",
+)
+
 _logger = logging.getLogger(__name__)
 
 _DEFAULT_BUILD_TIMEOUT_SECS = 1800  # 30 minute build timeout
 
 
 def _wait_for_completion(
-    batch_client: client.BatchV1Api, job_name: str, deadline_secs: Optional[int] = None
+    batch_client: Any,  # has type BatchV1Api
```

Comment:
I don't love all the weak typing I had to introduce here, but it was the only way to stop mypy from complaining. Adding a `# type: ignore` still left an issue with `read_namespaced_job_status` (used just below) supposedly not being defined. I left the types as comments to at least keep them documented.

Reply:
I think you can wrap the type in quotes to make it happy